src/components/NavigationBar.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Nav, Navbar, NavDropdown } from 'react-bootstrap';
+import { Link, NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { faHtml5, faCss3, faJs } from '@fortawesome/free-brands-svg-icons';
+
+const NavigationBar = () => {
+  return (
+    <Navbar collapseOnSelect expand="lg" bg="primary" variant="dark">
+      <Navbar.Brand as={Link} to="/">
+        CYF Quiz
+      </Navbar.Brand>
+      <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+      <Navbar.Collapse id="responsive-navbar-nav">
+        <Nav>
+          <Nav.Link as={NavLink} to="/" className="ml-5">
+            <FontAwesomeIcon
+              className="mr-2"
+              aria-hidden="true"
+              aria-label="Home icon"
+              icon={faHome}
+            />
+            Home
+          </Nav.Link>
+          <Nav.Link as={NavLink} to="/about" className="ml-5">
+            About
+          </Nav.Link>
+          <NavDropdown
+            title="Quizzes"
+            id="collasible-nav-dropdown"
+            className="ml-5"
```

Comment:
I don't think we need to change the spacing of the `Navbar` links (About and Quizzes). A lot of websites have the links right next to each other so I think that would be fine here too, and it would also stop the links from indenting when you open the menu on a small screen